src/Utils.cpp

```diff
@@ -805,13 +805,28 @@ bool CTable::SetCell(const CString& sColumn, const CString& sValue,
     return true;
 }
 
-bool CTable::GetLine(unsigned int uIdx, CString& sLine) const {
```

Comment:
Please add some test.

Reply:
I'm not really sure what kind of test would be appropriate (there are no test for anything CTable related currently). Anything you have in mind?

Reply:
I'm not sure how valueable tests are considering the CI is almost always red and takes forever to run 🤷‍♂️